pkg/option/runtime_options.go

```diff
@@ -43,6 +44,11 @@ var (
 		Description: "Enable debugging trace statements for load balancer",
 	}
 
+	specXDPTracing = Option{
+		Define:      "XDP_TRACING",
```

Comment:
Would setting the option trigger `bpf_xdp` recompilation + reload?

Reply:
Yes it does, `XDP_TRACING` now appears as a runtime option in `cilium config`, setting it to enabled triggers a recompilation. Running with `--debug-verbose=datapath` will cause this runtime option to [be enabled](https://github.com/dylandreimerink/cilium/blob/44d7a32426efadc08924e2f27081ebd470d243f0/daemon/cmd/daemon_main.go#L1362) at startup.